pages/portfolio.html

```diff
@@ -4,9 +4,71 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="../styles/portfolio.css">
+  
 </head>
-<body>
+ 
+<body>   
+  <header class = "header-portfolio">
+      <nav class = "nav-wrapper">
+        <div class = "title">
+        <h2>My Projects</h2>
+        </div>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./portfolio.html">Portfolio></a></li>
+          <li><a href="./about.html">About Me</a></li>
+            
+        </ul>
+      </nav>
+  
+  </header>
+
+
+
+  <section class="container">
+     
+     
+      <div class="box box-1"></div>
+      <div class="box box-2">
+        <a href="https://perugia33.github.io/Bello_Mundo_Project/"><H3> Bello Mundo -  A Photography Portfolio Website</H3></a> 
```

Comment:
you can add the target attribute to your a tag to open the link in a new tab. This helps the user stay on your page. https://www.w3schools.com/tags/att_a_target.asp